lib/services/sqlite_database.dart

```diff
@@ -44,91 +44,49 @@ FutureOr<void> _createTables(Database db, int prev, int next) async {
       keyword STRING UNIQUE NOT NULL
     );
   """);
-  
-   await db.execute("""
+
+  await db.execute("""
     CREATE TABLE IF NOT EXISTS $NOTIFICATION_TABLE (
       mail_piece_id STRING  NOT NULL REFERENCES $MAIL_PIECE_TABLE(id),
-      subscription_keyword STRING  NOT NULL REFERENCES $NOTIFICATION_SUBSCRIPTION_TABLE(keyword) ON DELETE CASCADE,
-      isCleared BIT DEFAULT 0 NOT NULL
+      subscription_keyword STRING  NOT NULL REFERENCES $NOTIFICATION_SUBSCRIPTION_TABLE(keyword) ON DELETE CASCADE
     );
   """);
 
   if (prev <= 1) {
-    try {
-      await db.execute("""
+    await db.execute("""
         ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN scanImgCID TEXT;
       """);
-    } catch (e) {
-      // Do nothing, prevent duplicate column from being added
-    }
   }
 
   if (prev <= 2) {
-    try {
-      await db.execute("""
-        ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN links TEXT;
-      """);
-      await db.execute("""
-        ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN emails TEXT;
-      """);
-      await db.execute("""
-        ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN phone_numbers TEXT;
-      """);
-      await db.execute("""
+    await db.execute("""
         ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN image_bytes TEXT;
       """);
-      await db.execute("""
+    await db.execute("""
         ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN featured_html TEXT;
       """);
-    } catch (e) {
-      // Do nothing, prevent duplicate column from being added
-    }
   }
 
-  if (prev <= 1) {
-    try {
-      await db.execute("""
+  if (prev <= 3) {
+    await db.execute("""
       ALTER TABLE $MAIL_PIECE_TABLE ADD COLUMN uspsMID TEXT;
```

Comment:
I'm getting a duplicate column here I believe. Any reason not to keep the try catch?

Reply:
Yeah, the reason you're getting that is because the previous migration wasn't handled properly. It left the database in a bad state, which means you'll probably need to reset your emulator. 

Ignoring the exceptions in the first place was the reason why it wasn't apparent that the migration logic was broken. (The solution was to ignore the error, rather than address the actual problem.)

I'll add back in the try/catch so that we can just finish this project.

Reply:
Oh ok sorry that makes perfect sense. I'll approve